dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/get-fee-record-correction-request-review.api-test.ts

```diff
@@ -0,0 +1,159 @@
+import { HttpStatusCode } from 'axios';
+import {
+  Bank,
+  FEE_RECORD_STATUS,
+  FeeRecordCorrectionTransientFormDataEntityMockBuilder,
+  FeeRecordEntityMockBuilder,
+  RECONCILIATION_IN_PROGRESS,
+  RECORD_CORRECTION_REASON,
+  RecordCorrectionTransientFormData,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import { withSqlAndMongoIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { testApi } from '../../../test-api';
+import { SqlDbHelper } from '../../../sql-db-helper';
+import { aBank } from '../../../../test-helpers';
+import { mongoDbClient } from '../../../../src/drivers/db-client';
+import { wipe } from '../../../wipeDB';
+
+console.error = jest.fn();
+
+const BASE_URL = '/v1/utilisation-reports/:reportId/fee-records/:feeRecordId/correction-request-review/:userId';
+
+describe(`GET ${BASE_URL}`, () => {
+  const getUrl = (reportId: number | string, feeRecordId: number | string, userId: string) => {
```

Comment:
could this be used anywhere else? if so it could be a test helper?

Reply:
I don't think it makes sense to do so because it is extremely dependent on the BASE_URL which will be different for each api-test file. There might be some urls that happen to have the same parameters but then we would have to pass in BASE_URL as well and have lots of variations for each different set of parameters. I think best to leave it as is as probably not worth the time it would take to commonise.